bndtools.core/src/bndtools/wizards/project/ImportBndWorkspaceWizard.java

```diff
@@ -0,0 +1,451 @@
+        if (Files.exists(settings)) {
+            Files.walkFileTree(settings, new SimpleFileVisitor<Path>() {
+                @Override
+                public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+                    Files.delete(file);
+                    return FileVisitResult.CONTINUE;
+                }
+
+                @Override
+                public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOException {
+                    Files.delete(dir);
+                    return FileVisitResult.CONTINUE;
+                }
+            });
+        }
+
+        final Path project = projectPath.resolve(".project");
+        Files.deleteIfExists(project);
+        final Path classpath = projectPath.resolve(".classpath");
+        Files.deleteIfExists(classpath);
+    }
+
+    private boolean importProjects(File rootImportPath, IProgressMonitor monitor) throws Exception {
+        Workspace bndWorkspace = Workspace.getWorkspace(rootImportPath);
+
+        int steps = bndWorkspace.getAllProjects().size() + 2;
+
+        monitor.beginTask("Importing Bnd workspace", steps);
+
+        importConfigurationProject(rootImportPath, monitor);
+
+        // Import Projects
+        for (Project bndProject : bndWorkspace.getAllProjects()) {
+            importBndProject(bndProject, bndWorkspace, monitor);
+        }
+
+        // build
+        monitor.subTask("Building workspace");
+        ResourcesPlugin.getWorkspace().build(IncrementalProjectBuilder.CLEAN_BUILD, monitor);
+        monitor.worked(1);
+        monitor.done();
+
+        return true;
+    }
+
+    private void importConfigurationProject(File rootImportPath, IProgressMonitor monitor) throws Exception {
+        monitor.subTask("Import configuration project 'cnf'.");
+        final IWorkspace eclipseWorkspace = ResourcesPlugin.getWorkspace();
+        final IWorkspaceRoot eclipseWorkspaceRoot = eclipseWorkspace.getRoot();
+        final Workspace bndWorkspace = Workspace.getWorkspace(rootImportPath);
+
+        // Prepare Eclipse Workspace
+        updateEclipseWorkspaceSettings(bndWorkspace);
+
+        // create generic project
+        final IProjectDescription cnfProjectDescription = eclipseWorkspace.newProjectDescription(Workspace.CNFDIR);
+        final IProject project = eclipseWorkspaceRoot.getProject(Workspace.CNFDIR);
+
+        deleteOldProjectFiles(Paths.get(bndWorkspace.getBase().toURI()).resolve(Workspace.CNFDIR));
+
+        // create JavaProject
+        IPath path = URIUtil.toPath(bndWorkspace.getBuildDir().toURI());
+        if (Platform.getLocation().isPrefixOf(path)) {
+            cnfProjectDescription.setLocation(null);
+        } else {
+            cnfProjectDescription.setLocation(path);
+        }
+
+        project.create(cnfProjectDescription, monitor);
+        project.open(monitor);
+
+        project.setDefaultCharset("UTF-8", monitor);
```

Comment:
If this is a new project lets set the charset to UTF-8.  But if this is an existing project (already have a .project) we should probably leave it so we don't wipe out the user's charset settings.